lib/collections/collections.js

```diff
@@ -425,7 +425,7 @@ var settings = new SimpleSchema ({
     radarrQUALITYPROFILEID: {
         type: Number,
         label: "Download quality profile for Radarr",
-        defaultValue: 1,
+        defaultValue: '1',
```

Comment:
Give me a bit to get home so i can change that and the corresponding check in radarrMethods.js